publisher-sdk/src/main/java/com/criteo/publisher/BidManager.java

```diff
@@ -230,23 +264,23 @@ private boolean killSwitchEngaged() {
     return config.isKillSwitchEnabled();
   }
 
-  private class CdbListener implements CdbCallListener {
-
-    @Override
-    public void onCdbRequest(@NonNull CdbRequest request) {
-      bidLifecycleListener.onCdbCallStarted(request);
-    }
+  /**
+   * Implementation specific to listening Cdb calls for updating the cache only
+   */
+  private class CdbCallForCacheListener extends CdbCallListener {
```

Comment:
It could be once we migrate all the integration modes to the live strategy. As you can see the live listener has the additional responsibility to pass the `BidResponse` through the `BidListener`, so all integration modes need to be ready to receive bids via this `BidListener`.